### ISSUE TITLE
error 403

### ISSUE THREAD

**Issue by yuekatoang** [fal-ai/sdxl-lightning-demo-app#8]
keep getting

POST /api/proxy 403

**Comment by dplakon** [comment]
same

**Comment by Adhamhossam000** [comment]
⨯ TypeError: fetch failed
    at node:internal/deps/undici/undici:12344:11
    at process.processTicksAndRejections (node:internal/process/task_queues:95:5) {
  cause: ConnectTimeoutError: Connect Timeout Error
      at onConnectTimeout (node:internal/deps/undici/undici:7492:28)
      at node:internal/deps/undici/undici:7448:50
      at Immediate._onImmediate (node:internal/deps/undici/undici:7478:37)
      at process.processImmediate (node:internal/timers:478:21)
      at process.callbackTrampoline (node:internal/async_hooks:130:17) {
    code: 'UND_ERR_CONNECT_TIMEOUT'
  }
}

**Comment by AviPeltz** [comment]
yup, looks like the import { route } from "@fal-ai/serverless-proxy/nextjs"; and fal-ai/serverless-client is deprecated so gotta swap out for the update "fal-ai/client"

**Comment by cjkihl** [comment]
Issue is still not fixed. Example by Fal AI has not been working for monts and code is old. Such a Joke.
